scripts/functions.sh

```diff
@@ -57,6 +52,9 @@ function clusterIsReady() {
     fi
 }
 
+MANIFEST_FILE="flow-capture.yml"
```

Comment:
If you manipulate the file the inject.sh script will not work anymore :thinking: 

The goal is to embedd these yamls into variables and have a single shell as oc / krew plugin

Reply:
I did update the PR to read the yaml after been updated by inject.sh lmk if that works